www-servers/automx2/automx2-2019.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Email client autoconfiguration service"
+HOMEPAGE="https://automx.org/"
+SRC_URI="https://gitlab.com/automx/automx2/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="acct-user/automx2
+	$(python_gen_cond_dep \
+		'>=dev-python/flask-migrate-2.5.2[${PYTHON_MULTI_USEDEP}]' python3_{7}
```

Comment:
Also why are these in BDEPEND. Why would build-machine acct-user/automx2? Granted I haven't tested this ebuild, but this **looks** weird to me.

Reply:
You probably have a point here; acct-* is not really needed before the software is run.